website/payments/models.py

```diff
@@ -218,7 +219,10 @@ class BankAccount(models.Model):
     valid_from = models.DateField(verbose_name=_("valid from"), blank=True, null=True,)
 
     valid_until = models.DateField(
-        verbose_name=_("valid until"), blank=True, null=True,
+        verbose_name=_("valid until"),
+        blank=True,
+        null=True,
+        help_text="Users can revoke the mandate at any time, as long as they do not have any Thalia Pay payments that have not been processed. If you revoke a mandate, make sure to check that all unprocessed Thalia Pay payments are paid in an alternative manner.",
```

Comment:
```suggestion
        help_text=_("Users can revoke the mandate at any time, as long as they do not have any Thalia Pay payments that have not been processed. If you revoke a mandate, make sure to check that all unprocessed Thalia Pay payments are paid in an alternative manner."),
```

Reply:
Didn't we stop translations?

Reply:
Yes we removed the translations, but that does not mean we can't just use the translation helper.